docs/UserGuide.md

```diff
@@ -202,6 +202,8 @@ Here is a reference table that briefly summarizes available commands:
 | `delete`    | Removes a client from the system.                      |
 | `list`      | Displays all clients currently stored in the system.   |
 | `filter`    | Filters clients based on specified criteria            |
+| `view`      | Opens a split view showing detailed client information |
+| `close`     | Closes the split view of client details               |
```

Comment:
Perhaps we can standardize whether we want to use `client details` or `detailed client information`.

Reply:
I would say `detailed client information` is more befitting of this context.